src/Mod/Fem/feminout/convert2TetGen.py

```diff
@@ -1,13 +1,14 @@
 # (c) 2010 LGPL
```

Comment:
This file doesn't have a uniform license text block or a `__title__` block. Should we add it ?

Reply:
yes, this file is the only Python module in FEM not changed to be according to flake8. It was developed by the project founder Jürgen Riegel years ago. It is not used ATM, thus it can not really be tested. Thus I left the code untouchted. See https://github.com/berndhahnebach/FreeCAD_bhb/commits/master/src/Mod/Fem/feminout/convert2TetGen.py and https://github.com/berndhahnebach/FreeCAD_bhb/commits/9376138d0623b0cae33b030c8de7bf4ff992f9c9/src/Mod/Fem/convert2TetGen.py

Reply:
https://github.com/FreeCAD/FreeCAD/commits/master/src/Mod/Fem/feminout/convert2TetGen.py

https://github.com/FreeCAD/FreeCAD/commits/9376138d0623b0cae33b030c8de7bf4ff992f9c9/src/Mod/Fem/convert2TetGen.py